.github/workflows/deploy.yml

```diff
@@ -55,3 +55,12 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v2
```

Comment:
Historically, GitHub pages could have a few minutes between pushing/deploying and the content appearing. Is that still the case? If so, do we need to wait a bit before triggering a new crawl, so that it is definitely indexing the new content?

Reply:
Hmm... Good point. I'd hate to delay the job, so I'll have this trigger another job that sleep waits then curls

Reply:
That seems equivalent to just waiting in this job? Can you spell out why do you think it's worth splitting in more detail?

Reply:
Idealogically it's two different concerns.
The deploy job _has_ finished deploying. It should mark itself as complete and revel in a job well done.

Triggering an index of the deployment isn't the deployment. It's a post-deploy event.

Technically speaking I think the only reason to split it is that we disallow concurrency (one job at a time) so in the event of a PR merging during a deployment it'd have to wait for our timeout before being able to build.

Reply:
Ah, I see.  Okay. Hopefully we can tee it up to be two jobs within a single workflow (by moving the concurrency to be on the job), rather than having to chain two workflows?

Reply:
Lets' try that!

Reply:
Ok, done. And specifically we wait until the homepage contains the `GITHUB_SHA`